src/Components/NavBar.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export default function NavBar() {
+  return (
+    <div className="Navbar">
+      <NavLink to="/home" activeStyle={{fontWeight: "bold", color: "black"}} exact={true}>Home</NavLink>
```

Comment:
If you use a.active{font-weight:..} in CSS file, you don't have to write activeStyle={{fontWeight: "bold", color: "black"}}. NavLink automatically chooses to activate link.